velox/expression/fuzzer/ExpressionFuzzer.h

```diff
@@ -101,6 +103,9 @@ class ExpressionFuzzer {
     //   "array_sort(array(T),constant function(T,T,bigint)) -> array(T)"}
     std::unordered_set<std::string> skipFunctions;
 
+    std::unordered_map<std::string, std::shared_ptr<ExprTransformer>>
+        functionTransformers;
```

Comment:
I would call it `exprTransformers`, `functionTransformers` seem to indicate that function is going to be transformed instead of expression itself